js/apps/keycloak-server/package.json

```
@@ -3,7 +3,9 @@
   "type": "module",
   "scripts": {
     "start": "wireit",
-    "clear-data": "rm -r ./server/data"
+    "local": "wireit",
+    "clear-data": "rm -r ./server/data",
+    "clear-server": "rm -r ./server"
```

Never been a big fan of this naming convention, perhaps we can do the following?
```suggestion
    "delete-data": "rm -r ./server/data",
    "delete-server": "rm -r ./server"
```

WDYT?

---

totally fine with me!